### ISSUE TITLE
pipenv on python38 on windows does not install from Pipfile

### ISSUE THREAD

**Issue by jaxley** [pypa/pipenv#4214]
### Issue description

pipenv on Python 3.8 on MacOS has no problem with a very simple Pipfile with a single dependency on the requests module (file contents below)

However, a user tried to run `pipenv install` on Windows with Python 3.8, and pipenv completes without installing anything.

I was curious if pipenv was properly interpreting the dependency graph, but running `pipenv graph` output nothing at all!  So, somehow it's incorrectly thinking there are 0 dependencies.

### Expected result

```
C:\myfolder pipenv install
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✔ Success! 
Updated Pipfile.lock (fbd99e)!
Installing dependencies from Pipfile.lock (fbd99e)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 5/5 — 00:00:02
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
```

### Actual result

```
C:\myfolder pipenv install
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✔ Success! 
Updated Pipfile.lock ()!
Installing dependencies from Pipfile.lock ()…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/0 — 00:00:02
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
```

### Steps to replicate

On Windows:
1. git clone repo containing python files (includes *only* Pipfile, NOT Pipfile.lock)
2. python -m pip install pipenv
3. pipenv install

I've ensured that both `setuptools` and `pip` are the latest versions. Still doesn't change anything.

-------------------------------------------------------------------------------

TODO:  waiting on end user to provide the output. Will update.  Should be the latest version as of today

**Comment by techalchemy** [comment]
If you can obtain a copy of the `Pipfile` in question that would be helpful

**Comment by jaxley** [comment]
[Pipfile.txt](https://github.com/pypa/pipenv/files/4548868/Pipfile.txt)

**Comment by techalchemy** [comment]
Would you mind verifying the output of `pipenv --where`? I have tried to reproduce this on a fresh windows VM with python 3.8 and it worked ok:

```console
PS C:\Users\hawkerz\git\test> type Pipfile
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]

[packages]
requests = "*"

[requires]
python_version = "3.8"
PS C:\Users\hawkerz\git\test> pipenv install
Creating a virtualenv for this project…
Pipfile: C:\Users\hawkerz\git\test\Pipfile
Using C:/Program Files/Python38/python.exe (3.8.2) to create virtualenv…
[  ==] Creating virtual environment...created virtual environment CPython3.8.2.final.0-64 in 3055ms

  creator CPython3Windows(dest=C:\Users\hawkerz\.virtualenvs\test-J9Gav-PO, clear=False, global=False)

  seeder FromAppData(download=False, pip=latest, setuptools=latest, wheel=latest, via=copy, app_data_dir=C:\Users\hawkerz\AppData\Local\pypa\virtualenv\seed-app-data\v1.0.1)

  activators BashActivator,BatchActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator

Successfully created virtual environment!
Virtualenv location: C:\Users\hawkerz\.virtualenvs\test-J9Gav-PO
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
           Building requirements...
Resolving dependencies...
Success!
Updated Pipfile.lock (fbd99e)!
Installing dependencies from Pipfile.lock (fbd99e)…
  ================================ 5/5 - 00:00:04
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
PS C:\Users\hawkerz\git\test> pipenv --version
pipenv, version 2020.4.1b1
```

**Comment by matteius** [comment]
@jaxley Please recheck with `2022.1.8` and there are some more windows fixes in `main` branch right now, if you find its still an issue report back or open a new ticket.  Thanks!
